### PR TITLE
feat(navigation): electron adjustments

### PR DESCRIPTION
**Pull request by dominikfeininger**

### CODE REVIEW

File: lib/NativeUtils.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ module.exports = class NativeUtils extends Utils {
         // TODO: add const variables in spec file and add path spec, remove "tbd"
         android: 'file:///android_asset/www/index.html',
         ios: `${this.context.getUrl()}`,
-        electron: `${this.context.getUrl()}`,
```
Comment (vobu) [inline lib/NativeUtils.js:14]: probably add a comment why `electron` is missing here?

File: lib/NativeUtils.js [hunk 2]
```diff
@@ -40,8 +39,6 @@ module.exports = class NativeUtils extends Utils {
                 this.path.currentPath = this.path.android;
             } else if (this.getConfig('platform') === 'ios') {
                 this.path.currentPath = this.path.ios;
-            } else if (this.getConfig('platform') === 'electron') {
```
Comment (vobu) [inline lib/NativeUtils.js:43]: same here: probably add a comment why `electron` is missing here

File: lib/BrowserUtils.js [hunk 3]
```diff
@@ -3,10 +3,15 @@ const fs = require('fs');
 const logger = require('./Logger');
 const path = require('path');
 const Utils = require('./Utils');
+const wdioUi5 = require('./wdioUi5-index')
 
 // Singleton
 module.exports = class BrowserUtil extends Utils {
     static _instance;
+    path = {
```
Comment (vobu) [inline lib/BrowserUtils.js:11]: would also be helpful to have a comment here why `electron` is part of BrowserUtils.js (and not NativeUtils)

File: lib/NativeUtils.js (hunk 1 above)
Comment (dominikfeininger) [reply lib/NativeUtils.js:14]: // electron is part of ./BrowserUtils.js

File: lib/NativeUtils.js (hunk 2 above)
Comment (dominikfeininger) [reply lib/NativeUtils.js:43]: // electron is part of ./BrowserUtils.js

Comment (dominikfeininger) [reply lib/NativeUtils.js:43]: // electron is part of ./BrowserUtils.js

File: lib/BrowserUtils.js (hunk 3 above)
Comment (dominikfeininger) [reply lib/BrowserUtils.js:11]: I am going to write it in Readme.

**Pull request merged**